media-video/shotcut/shotcut-19.12.31.ebuild

```diff
@@ -49,6 +49,12 @@ src_prepare() {
 
 	sed -i -e '/QT.*private/d' \
 		src/src.pro || die
+	REPLACE_PATTERN="Shotcut is a free, open source, cross-platform video editor."
+	sed -i \
+		-e "s/^Comment=Video Editor$/Comment=$REPLACE_PATTERN"/ \
+		-e '/^Comment\[fr\]=/d' \
+		-e '/^Comment\[ru\]=/d' \
+		packaging/linux/org.shotcut.Shotcut.desktop || die
```

Comment:
I don't think it's worth it to fix the previous version.

Reply:
More importantly, changing installed files would require a revbump. And that's pointless considering 20.06.28 bump.